css/modules/icons.scss

```diff
@@ -1,28 +1,59 @@
-#block-searchform .button {
-    width:0; height:1em;
-    overflow: hidden;
-    text-indent: -10em;
-    background-repeat:no-repeat;
-    background-position: center center;
-    background-size:($size-gutter*2) ($size-gutter*2);
-    background-image:url('images/fa-svg/white/search.svg');
-}
-#nav1 {
-    a {
-        background-size:24px;
-        background-repeat:no-repeat;
-        background-position:$size-gutter center;
-        padding:0 0 0 52px;
-        font-size:$font-size-big;
-        line-height:4rem;
+header {
```

Comment:
We *can* continue to rely on the id for the #block-searchform.  That's put in by our own custom theme.  It might be a little clearer in the future to still see that selector, instead of "header".